Cargo.toml

```diff
@@ -10,7 +10,7 @@ license = "MIT"
 readme = "README.md"
 
 [dependencies]
-avro-rs = "0.13"
+apache-avro = {path="/home/martin/git/apache/avro/lang/rust/avro"}
```

Comment:
Unfortunately there is no official release yet and using `git` dependency reference does not work (https://github.com/flavray/avro-rs/pull/99#issuecomment-1018203577), so you will need to clone the project locally and adapt the path.

Reply:
Sadly this would prevent the crate from being published.

An alternative could be to have `apache-avro` as a git submodule and include it as part of a cargo workspace.
This would be temporary until `apache-avro` gets released.
However it would be quite a breaking change for consumer of this library as it would generate types specifically for `apache-avro` (which wouldn't be available on crates.io but only in the re-exported module of this crate...).

Do you have an estimate date for an official (even beta) release of `apache-avro` ?

Reply:
Anytime now...
I think the release manager will do it next week!

Reply:
Hey @martin-g, I wanted to check in on the status of this effort. I'm looking to use both `rsgen-avro` and the `apache-avro` crate (without having to rely on `avro-rs`). Is there anything in particular blocking the release of the `apache-avro` crate?

Reply:
The only problem is that Apache Avro releases all SDKs at once and the release manager didn't have time to do it yet...
I will ask whether it is OK to release just the Rust SDK. But then I expect a problem in finding 3 PMC members to vote on it. It is just me who works on the Rust SDK ...

The only "solution" I could offer you is to use it as a Git dependency (https://github.com/apache/arrow-datafusion/pull/2784/files#diff-d37870d03a10eb84378f2ba0e25954e824b92ea69fcde2db625b32b0f6e4851aR41)

Reply:
Avro 1.11.1 is being voted - https://lists.apache.org/thread/8tk92x804owmjbvtj57xxzwhxn1qh3ho!
I'd be thankful if you can test your project(s) and report any issues!

Reply:
I'm not subscribed to the mailing list at this moment (and can also not submit binding votes), but I just quickly tried using tag `release-1.11.1-rc1` on this branch and in our own internal project, and in both cases all tests appear to be passing!

Just to confirm, `release-1.11.1-rc1` would also ensure that apache-avro would be published to crates.io by the release manager, correct?

Reply:
Correct!
Avro Rust SDK will be versioned as 0.14.0 though.

Reply:
Awesome, thanks for the heads up!

Reply:
Hey! Did the release manager forget the rust package on crates.io ? I see the releases in javascript/python/etc. but not the rust one.